pkg/network/tracer/tracer_linux_test.go

```diff
@@ -50,13 +50,8 @@ import (
 	manager "github.com/DataDog/ebpf-manager"
 )
 
-var kv470 kernel.Version = kernel.VersionCode(4, 7, 0)
-var kv kernel.Version
-
-func setKernelVersion() (err error) {
-	kv, err = kernel.HostVersion()
-	return
-}
+var kv470 = kernel.VersionCode(4, 7, 0)
+var kv = kernel.MustHostVersion()
```

Comment:
back then @hmahmood was against not reporting the error

Reply:
In production code, I wouldn't panic, but this is in tests and will happen immediately.

Reply:
`panic` works too :-)